### PR TITLE
FUSETOOLS-2056 - Use System File separator

### PR DESCRIPTION
**Pull request by apupier**

### CODE REVIEW

File: editor/plugins/org.fusesource.ide.projecttemplates/src/org/fusesource/ide/projecttemplates/maven/CamelProjectConfigurator.java [hunk 1]
```diff
@@ -132,7 +133,7 @@ private void updateMappings(IPath[] paths, IProject project, IVirtualFolder vFol
 			IVirtualResource[] mappings = ComponentCore.createResources(srcFolder);
 			boolean found = false;
 			for (IVirtualResource mapping : mappings) {
-				if (mapping.getProjectRelativePath().toOSString().equals("/" + srcFolder.getProjectRelativePath().toOSString())) {
+				if (mapping.getProjectRelativePath().toOSString().equals(File.separator + srcFolder.getProjectRelativePath().toOSString())) {
 					mapping.createLink(new Path("/").append(sourceLoc), 0, null);
```
Comment (lhein) [inline editor/plugins/org.fusesource.ide.projecttemplates/src/org/fusesource/ide/projecttemplates/maven/CamelProjectConfigurator.java]: Wondering if we need File.separator here too

**Comment by lhein** [comment]
applied manually. thx Aurelien

File: editor/plugins/org.fusesource.ide.projecttemplates/src/org/fusesource/ide/projecttemplates/maven/CamelProjectConfigurator.java (hunk 1 above)
Comment (apupier) [inline editor/plugins/org.fusesource.ide.projecttemplates/src/org/fusesource/ide/projecttemplates/maven/CamelProjectConfigurator.java]: I was wondering too. Seems that we don't need, it is to be relative to project path and is handled by Eclipse correctly

Comment (robstryker) [inline editor/plugins/org.fusesource.ide.projecttemplates/src/org/fusesource/ide/projecttemplates/maven/CamelProjectConfigurator.java]: This seems like File.separator might not be the right thing to do here.   In fact it's better not to use the .toOSString at all...  It'd probably be better to do 

if (mapping.getProjectRelativePath().equals(srcFolder.getProjectRelativePath().makeAbsolute())).  On windows, changing an IPath to the OSString may end up with something like C:/some/relative/path  which is wrong.  I'd suggest you test this on windows before making this change.

**Pull request closed without merging**